seed/models/analyses.py

```diff
@@ -45,3 +51,14 @@ class Analysis(models.Model):
     # that are applicable to the entire analysis (ie all properties involved).
     # For property-specific results, use the AnalysisPropertyView's parsed_results
     parsed_results = JSONField(default=dict, blank=True)
+
+    def getPropertyViewInfo(self, property_id=None):
```

Comment:
Missed this first time around, but SEED uses snake-case for all python variable and function names. If you have a minute, could you update our tox.ini for flake8 to install pep8-naming? https://stackoverflow.com/questions/28725116/flake8-doesnt-report-mixed-case-function-names

Reply:
AFAIK this should be working.